### ISSUE TITLE
Caused by java.lang.NullPointerException - com.onesignal.WebViewManager.stopped (WebViewManager.java:320)

### ISSUE THREAD

**Issue by renatobentorocha** [OneSignal/react-native-onesignal#1201]
**Description:**

* NullPointerException pointer exception occurring on com.onesignal.WebViewManager.stopped (WebViewManager.java:320)
* Error from crashlytics

**Environment**

```
❯ npx react-native info                                   
info Fetching system and libraries information...
System:
    OS: macOS 11.2.3
    CPU: (8) x64 Apple M1
    Memory: 23.50 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.16.0 - ~/.nvm/versions/node/v14.16.0/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v14.16.0/bin/npm
    Watchman: 4.9.0 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.10.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    Android SDK:
      API Levels: 28, 29, 30
      Build Tools: 28.0.3, 29.0.2, 30.0.3
      System Images: android-23 | Google APIs ARM EABI v7a, android-29 | Intel x86 Atom_64, android-29 | Google Play Intel x86 Atom, android-30 | Google APIs Intel x86 Atom, android-S | Google Play ARM 64 v8a
      Android NDK: Not Found
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.7199119
    Xcode: 12.4/12D4e - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_282 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.4 => 0.63.4
```

**Steps to Reproduce Issue:**

We couldn't be able to reproduce this crash. But there are several occurrences on crashlytics.

**Anything else:**

** Stacktrace

```
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
       at com.onesignal.WebViewManager.stopped(WebViewManager.java:320)
       at com.onesignal.ActivityLifecycleHandler.onActivityStopped(ActivityLifecycleHandler.java:110)
       at com.onesignal.ActivityLifecycleListener.onActivityStopped(ActivityLifecycleListener.java:103)
       at android.app.Application.dispatchActivityStopped(Application.java:515)
       at android.app.Activity.dispatchActivityStopped(Activity.java:1357)
       at android.app.Activity.onStop(Activity.java:2499)
       at androidx.fragment.app.FragmentActivity.onStop(FragmentActivity.java:558)
       at androidx.appcompat.app.AppCompatActivity.onStop(AppCompatActivity.java:216)
       at android.app.Instrumentation.callActivityOnStop(Instrumentation.java:1474)
       at android.app.Activity.performStop(Activity.java:8195)
       at android.app.ActivityThread.callActivityOnStop(ActivityThread.java:5117)
       at android.app.ActivityThread.performStopActivityInner(ActivityThread.java:5090)
       at android.app.ActivityThread.handleStopActivity(ActivityThread.java:5170)
       at android.app.servertransaction.StopActivityItem.execute(StopActivityItem.java:41)
       at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2267)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:237)
       at android.app.ActivityThread.main(ActivityThread.java:8167)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)
```

**100% occur in background**

![](https://user-images.githubusercontent.com/18239349/115280011-ccfa2080-a11d-11eb-9537-f5d5d4203cbc.png)

**Comment by rgomezp** [comment]
Howdy,
What version of the SDK are you using?

**Comment by renatobentorocha** [comment]
> Howdy,
> What version of the SDK are you using?

"react-native-onesignal": "4.0.4",

**Comment by rgomezp** [comment]
Howdy,
Thanks for the info.

Seems to be coming from [here](https://github.com/OneSignal/OneSignal-Android-SDK/blob/4.2.0/OneSignalSDK/onesignal/src/main/java/com/onesignal/WebViewManager.java#L320).

It looks like the currentActivityName _can_ be null.

We will fix this as soon as possible. 

Thanks again

**Comment by rgomezp** [comment]
Update. This has been resolved in 4.0.7. Please update to get the fix.

Enjoy!

**Comment by renatobentorocha** [comment]
> Update. This has been resolved in 4.0.7. Please update to get the fix.
> 
> Enjoy!

Thanks!
